CHANGES.txt

```diff
@@ -2,10 +2,10 @@
 CHANGELOG
 #########
 
-3.6.0 (unreleased)
+3.5.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Fix wrong localizer usage when deactivated. (#505)
```

Comment:
What means `wrong` here?

Reply:
I've no idea, actually I don't really know what this localizer is about.
What I understood is that it is trying to read the request params and in case they are wrong it fails without handling the error properly.